<launch>
  <arg name="robotName" default="valkyrie" />
  <arg name="mode" default="passthrough" />
  <arg name="imuSensor" default="pelvis_imu_sensor_at_pelvis_frame" />

  <node name="ros2lcm_ihmc" type="ros2lcm_ihmc" pkg="oh_translators" output="screen" args="$(arg mode) $(arg robotName) $(arg imuSensor)" />
  <node name="lcm2ros_ihmc" type="lcm2ros_ihmc" pkg="oh_translators" output="screen" args="$(arg robotName)"/>

  <node name="ros2lcm_camera" type="ros2lcm_camera" pkg="oh_translators" output="screen">
  	<param name="camera_topic" type="str" value="/multisense/left/image_rect_color" />
  	<param name="flip_rgb" type="bool" value="false" />
  </node>

  <node name="ros2lcm_nasa" type="ros2lcm_nasa" pkg="oh_translators" output="screen"/>
  <node name="lcm2ros_nasa" type="lcm2ros_nasa" pkg="oh_translators" output="screen"/>
</launch>
